src/pages/dashboard/dashboard.scss

```diff
@@ -65,20 +71,24 @@ ul.chart-legend {
   padding: 0 5px;
 }
 
+$small-multiples-chart-bp-sm: 480px;
```

Comment:
there are predefined screen size breakpoints in `/src/scss/breakpoints.scss` let's use those instead.

Reply:
@goleary revised in 4e1978a.